Assets/Scripts/Hazard.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Hazard : MonoBehaviour
+{
+    private AudioSource audioSource;
+
+    private void Start()
+    {
+        audioSource = GetComponent<AudioSource>();
+    }
+
+    private void OnTriggerEnter2D(Collider2D collision)
+    {
+
+        if (collision.gameObject.CompareTag("Player"))
+        {
+            Debug.Log("Player entered hazard!");
+            audioSource.Play();
+            //PlayerCharacter player = collision.GetComponent<PlayerCharacter>();
+            //player.transform.position.Set(0, 0, 45);
+            
+            //SceneManager.LoadScene(SceneManager.GetActiveScene().name);
+        }
+        //else
```

Comment:
Again, maybe take out the extra code and put it in a google doc since it isn’t being used?